handler/auth.go

```diff
@@ -81,6 +81,7 @@ func (h *Handler) Register(w http.ResponseWriter, r *http.Request) {
 		log.Printf("error sending registration email to user %s: %s", user.Username, err.Error())
 	}
 
+	w.Header().Add("Content-Type", "application/json")
```

Comment:
But there's no content...?

Reply:
I got a little happy with the content types, whoops